_includes/footer.html

```diff
@@ -12,7 +12,7 @@
         <p>Thanks to <a class="footer__links" href="https://ti.to">Tito</a> for supporting EmberFest <br/>Graphics from the Noun Project and Freevector</p>
       </div>
     <div class="footer__bottomNav">
-      <a class="footer__links" href="https://cfp.emberfest.eu/events/emberfest-2021">CFP</a> • <a class="footer__links" href="/sponsorship.pdf" download>Sponsorship</a> • <a class="footer__links" href="/code-of-conduct">Code of Conduct</a> • <a class="footer__links" href="/cancellation">Cancellation</a> • <a class="footer__links" href="/privacy">Privacy</a> • <a class="footer__links" href="/imprint">Imprint</a>
+      <a class="footer__links" href="https://cfp.emberfest.eu/events/emberfest-2021">CFP</a> • <a class="footer__links" href="/sponsorship.pdf" download>Sponsorship</a> • <a class="footer__links" href="/code-of-conduct">Code of Conduct</a> • <a class="footer__links" href="corona-safety/">Corona Virus & Safety</a> • <a class="footer__links" href="/cancellation">Cancellation</a> • <a class="footer__links" href="/privacy">Privacy</a> • <a class="footer__links" href="/imprint">Imprint</a>
```

Comment:
```suggestion
      <a class="footer__links" href="https://cfp.emberfest.eu/events/emberfest-2021">CFP</a> • <a class="footer__links" href="/sponsorship.pdf" download>Sponsorship</a> • <a class="footer__links" href="/code-of-conduct">Code of Conduct</a> • <a class="footer__links" href="/corona-safety">Corona Virus & Safety</a> • <a class="footer__links" href="/cancellation">Cancellation</a> • <a class="footer__links" href="/privacy">Privacy</a> • <a class="footer__links" href="/imprint">Imprint</a>
```

Otherwise the link does not work on any page that’s not the index.